distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -118,6 +118,21 @@ public void startManagedService(String servicePid, Map<String, Object> propertie
         startManagedService(sourceConfig, properties);
     }
 
+    /**
+     * Stops a managed service.
+     *
+     * @param servicePid
+     * @throws IOException
+     */
+    public void stopManagedService(String servicePid) throws IOException {
```

Comment:
Not sure at all. Right now, I think this method should be considered experimental and untested.